[doc] Add example to cross-plugin-concepts (#12672)
elastic/logstash: docs/static/cross-plugin-concepts.asciidoc

whitespace-delimited URIs into separate entries. This behaviour enables the expansion
of an arbitrary list of URIs from a single Environment- or Keystore-variable.

> The following plugins and options support this functionality:

Reviewer: ```suggestion
These plugins and options support this functionality:
```

* <<plugins-inputs-elasticsearch-hosts,Elasticsearch input plugin - `hosts`>>
* <<plugins-outputs-elasticsearch-hosts,Elasticsearch output plugin - `hosts`>>
* <<plugins-filters-elasticsearch-hosts,Elasticsearch filter plugin - `hosts`>>

> For example, users use this functionality to define an environment variable with
> multiple whitespace-delimited URIs to use it for the options above, e.g.:
> 
> ```
> ES_HOSTS="es1.example.com es2.example.com:9201 es3.example.com:9201"
> ```

Reviewer: ```suggestion
You can use this functionality to define an environment variable with
multiple whitespace-delimited URIs and use it for the options above.

**Example**

```
ES_HOSTS="es1.example.com es2.example.com:9201 es3.example.com:9201"
```
```

> For example, users use this functionality to define an environment variable with
> multiple whitespace-delimited URIs to use it for the options above, e.g.:
> 
> ```
> ES_HOSTS="es1.example.com es2.example.com:9201 es3.example.com:9201"
> ```

Reviewer: ```suggestion
You can use this functionality to define an environment variable with
multiple whitespace-delimited URIs and use it for the options above.

**Example**
```
ES_HOSTS="es1.example.com es2.example.com:9201 es3.example.com:9201"
```
```

Reviewer: Did you overlook this suggestion or choose not to take it?  If you'd like to discuss, I can explain the thinking behind the suggestion. 
https://github.com/elastic/logstash/pull/12672#discussion_r577219716
